f3/granite.go

```diff
@@ -563,7 +566,8 @@ func (i *instance) tryConverge() error {
 	if i.phase != CONVERGE_PHASE {
 		return fmt.Errorf("unexpected phase %s, expected %s", i.phase, CONVERGE_PHASE)
 	}
-	timeoutExpired := i.host.Time() >= i.phaseTimeout
+	timeoutExpired := i.host.Time().After(i.phaseTimeout) || i.host.Time().Equal(i.phaseTimeout)
```

Comment:
I don't think we need to check the `Equal` case. The local time uses nanosecond precision so the probability of it happening is insignificant.

Reply:
I think it's relatively important not to rely on time passing between whatever host system triggered the alarm (potentially at exactly the right time) and reaching this code. I am going to guess that in simulation the clock has not moved forward by the time this executes.